sqlglot/parser.py

```diff
@@ -4793,10 +4793,15 @@ def _parse_star(self) -> t.Optional[exp.Expression]:
         return self._parse_placeholder()
 
     def _parse_parameter(self) -> exp.Parameter:
-        wrapped = self._match(TokenType.L_BRACE)
+        self._match(TokenType.L_BRACE)
+
         this = self._parse_var() or self._parse_identifier() or self._parse_primary()
```

Comment:
can this be parse_id_var or self._parse_primary()

Reply:
let me take a look

Reply:
or parse_field() handles it